esp/esp/program/models/__init__.py

```diff
@@ -1046,6 +1046,14 @@ def getShirtInfo(self):
 
     @cache_function
     def incrementGrade(self): 
+        """
+        Increments the effective school year of the program.
```

Comment:
This is not the behavior of the function. The function returns a value that the caller is responsible for adding to the school year. The function returns either 0 or 1, depending on the Tag increment_default_grade_level.